lib/components/Carousel.js

```diff
@@ -88,7 +88,7 @@ var Carousel = function (_Component) {
         };
```

Comment:
Please do not commit files in the lib folder as they only add noise to the review... I can generate them here before publishing to npm. Thanks